Sites/GD2/Project 1/index.html

```diff
@@ -0,0 +1,101 @@
+<!DOCTYPE html>
+
+<html <head>
```

Comment:
Your `html` element is missing its closing bracket:

```suggestion
<html>
 <head>
```